src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -93,6 +104,50 @@ export default {
         }
     },
     methods: {
+        setMoveIndex(index) {
+            this.moveIndex = index;
+        },
+        dropCondition(e) {
+            let targetIndex = Array.from(document.querySelectorAll('.c-c__drag-ghost')).indexOf(e.target);
+            if (targetIndex > this.moveIndex) { targetIndex-- }
```

Comment:
See comment above about using `event.dataTransfer.getData(...)` to get the moveIndex instead of using `this.moveIndex` here.

Reply:
N/A - leave as is for now.